app/controllers/standard_steps_controller.rb

```diff
@@ -21,8 +21,13 @@ def update
 
   private
 
+  # Hook for step controllers to add behavior after successful update. This is
+  # intentionally a no-op in the StandardStepsController.
```

Comment:
I'd prefer the method name be clearer than we add comments. 

What about `after_successful_update_hook`?

Reply:
Done.